Minor editorial nits (#2800)
httpwg/http-extensions: draft-ietf-httpbis-unprompted-auth.md

All of the byte sequences below are encoded using base64url (see {{Section 5 of

> !BASE64=RFC4648}}) without quotes and without padding. In other words, these

Reviewer: ```suggestion
!BASE64=RFC4648}}) without quotes and without padding. In other words, the
```

> byte-sequence authentication parameters' values MUST NOT include any characters

Reviewer: I prefer the second wording you mentioned, I'll incorporate it into this branch via suggestions.

```suggestion
values of these byte-sequence authentication parameters MUST NOT include any characters
```

other then ASCII letters, digits, dash and underscore.

The integer below is encoded without a minus and without leading zeroes. In

> other words, the integer authentication parameter's value MUST NOT include any

Reviewer: ```suggestion
other words, the value of this integer authentication parameter MUST NOT include any
```
